### PR TITLE
MD-585: change column nullability to display as readable string

### PR DESCRIPTION
**Pull request by cchang738**
Please review. Pre-commit test running at

http://10.10.104.91:8080/job/Framework%20Regression/241/console

Thanks.

### CODE REVIEW

**Comment by sudheeshkatkam** [comment]
+1

File: framework/src/main/java/org/apache/drill/test/framework/DrillTestJdbc.java [hunk 1]
```diff
@@ -251,32 +251,30 @@ private void executeLimitZeroQuery(String query) throws IOException, SQLExceptio
...
+        columnNullabilities.add(resultSet.getMetaData().isNullable(i));
       }
       
-      String msg = "\n" + query + "\n" + columnLabels + "\n"
-    		  + this.columnLabels + "\n"
-    		  + "\n" + Utils.getTypesInStrings(types) + "\n"
-    		  + Utils.getTypesInStrings(this.types) + "\n"
-    		  + "\n" + nullability + "\n"
-    		  + this.nullability + "\n";
+      String msg = "\n" + query + "\n" + columnLabels + "\n" + this.columnLabels + "\n"
```
Comment (sudheeshkatkam) [inline framework/src/main/java/org/apache/drill/test/framework/DrillTestJdbc.java]: Suggestion: can we add some descriptors to the message so we know which query columnLabels and columnNullabilities correspond to?

**Comment by cchang738** [comment]
new pre-commit run http://10.10.104.91:8080/job/Framework%20Regression/243/console

**Comment by cchang738** [comment]
Thanks for review.

**Pull request merged**